tests/virt_autotest/login_console.pm

```diff
@@ -168,7 +168,10 @@ sub login_to_console {
             use_ssh_serial_console;
             save_screenshot;
             #start upgrade
-            enter_cmd("DISPLAY= yast.ssh");
+            #enter_cmd("DISPLAY= yast.ssh");
+            #DIAPLAY= might be culprit that prevents host upgrade from proceeding at SCC registration. Please refer to bsc#1218798.
+            record_soft_failure("bsc#1218798 - [SLES][15-SP6][x86_64][Build46.40] Unable to create repository due to valid metadata not found");
```

Comment:
According to https://confluence.suse.com/display/qasle/Soft-Fails+Mitigation, better to avoid unconditional soft_fail. If it is not easy to script the conditions, please do remember to remove it once it's solved.

Reply:
I ever considered this to have a if condition to hold this soft failure, but chose to add soft failure directly here because it is only for offline host upgrade. But I can still add a if condition, if it is absolutely necessary.

Reply:
No... The conditional soft-fail means the conditions to tell that this specific bug is happening, not some fake conditions or some meaningless conditions to just say it is conditional ;-). For example, here for this specific case, if the key symptom of scc problem is happening, we claim the soft_fail by the bug. I know this is not that feasible for the case. So we can go with the latter case -- just remember to remove it once bug verified fixed.

I raise this comment mainly to remind the rules. Let's follow wherever possible.

Reply:
I already made the change to make the soft-failure more specific.